client/db/bolt/db.go

```diff
@@ -114,6 +114,21 @@ func (db *boltDB) Store(k string, v []byte) error {
 	})
 }
 
+// ValueExists checks if a value was previously stored in the general-use
+// bucket at the specified key.
+func (db *boltDB) ValueExists(k string) (bool, error) {
```

Comment:
A quick test in db_test.go would be good.  `client/core`'s tests hit it, but that's a different package.  A few lines in `TestStore` in db_test.go for the positive and negative cases would be fine.